.github/workflows/build.yml

```diff
@@ -41,7 +41,12 @@ jobs:
             /home/runner/.cache/Cypress
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
       - name: Install
-        run: yarn install --frozen-lockfile && tsc -v
+        run: |
+          corepack prepare yarn@3.5.0 --activate
```

Comment:
This line is causing build failure, maybe it's best to use a stable version of yarn